src/main/java/frc/robot/commands/SimpleDriveCommand.java

```diff
@@ -23,17 +21,16 @@ public class SimpleDriveCommand extends EntechCommandBase {
      * @param drive The drive subsystem on which this command will run
      * @param stick Driver joystick object
      */
-    public SimpleDriveCommand(DriveSubsystem drive, Joystick joystick, Supplier<Double> yawAngleSupplier) {
+    public SimpleDriveCommand(DriveSubsystem drive, Supplier<DriveInput> operatorInput) {
```

Comment:
This is the simplification I really like throughout the PR.

Reply:
yes! much more testable and flexible than passing the joystick directly around